ChangeLog

```diff
@@ -1,3 +1,7 @@
+Version 11.1.1
```

Comment:
If we're strictly doing semver this is 11.2.0 because it's a feature :) Not that we're that strict in this repo.